Azure/sonic-buildimage
# [SNMP] fix issue #1874, ERR snmpd[34]: Error allocating more space for arpcache. Cache will continue to be limited to 4096 entries

Fixed issue #1874



**- What I did**

* Merge https://sourceforge.net/p/net-snmp/code/ci/d6e7f8962b110a2cd65667b2c57a8a60ad3cd222/
* Using stg to apply patch

**- How I did it**

**- How to verify it**

1. Create 4K+ ARP entries
2. `snmpwalk -v2c -c public 127.0.0.1` in container `snmp`
3. Check the following log doesn't exist in `/var/log/syslog`

```
ERR snmpd[34]: Error allocating more space for arpcache. Cache will continue to be limited to 4096 entries
```

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
